src/aura/HH_Canvas/HH_Canvas.cmp

```diff
@@ -17,9 +17,9 @@
 
     <!-- markup follows... -->
 
-    <ul id="SortCanvas" class="slds-list--horizontal slds-has-cards--space" style="display: inline;">
+    <ul id="SortCanvas" class="slds-list--horizontal slds-has-dividers--around-space slds-wrap">
```

Comment:
I did do testing but only in Chrome, the inline style was acting wonky after the updated slds, it caused the card to take the entire width of the page. When I removed it, all was well with the size but then when I added a bunch of contacts it wasn't wrapping so I added the slds-wrap. slds-has-cards--space was deprecated so switched that over to the slds-has-dividers--around.